### PR TITLE
Update AArch64 features to Linux 6.3.

### PR DESCRIPTION
**Pull request by tamaszarm**

### CODE REVIEW

**Comment by Mizux** [comment]
LGTM found them at https://elixir.bootlin.com/linux/latest/source/arch/arm64/include/uapi/asm/hwcap.h#L84

File: include/internal/hwcaps.h [hunk 1]
```diff
@@ -83,6 +83,27 @@ CPU_FEATURES_START_CPP_NAMESPACE
 #define AARCH64_HWCAP2_ECV (1UL << 19)
 #define AARCH64_HWCAP2_AFP (1UL << 20)
 #define AARCH64_HWCAP2_RPRES (1UL << 21)
+#define AARCH64_HWCAP2_MTE3 (1 << 22)
```
Comment (Mizux) [inline include/internal/hwcaps.h:86]: missing `UL` after `1`

**Review by Mizux — CHANGES_REQUESTED** [review]

File: include/internal/hwcaps.h (hunk 1 above)
Comment (tamaszarm) [reply include/internal/hwcaps.h]: They should have it now.

**Review by Mizux — APPROVED** [review]
LGTM

**Pull request merged**